habitat/core/simulator.py

```diff
@@ -63,7 +63,11 @@ class Sensor:
 
     def __init__(self, *args: Any, **kwargs: Any) -> None:
         self.config = kwargs["config"] if "config" in kwargs else None
-        self.uuid = self._get_uuid(*args, **kwargs)
+        if hasattr(self.config, "UUID"):
+            # We allow any sensor config to override the UUID
+            self.uuid = self.config.UUID
+        else:
```

Comment:
@mathfac Actually, I am not sure I like this method because calling _get_uuid will return the wrong UUID.